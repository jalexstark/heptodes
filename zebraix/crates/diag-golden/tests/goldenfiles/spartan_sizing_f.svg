<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="300pt" height="450pt" viewBox="0 0 300 450">
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 0 337.5 L 0 112.5 M 300 337.5 L 300 112.5 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 0 112.5 L 300 337.5 M 300 112.5 L 0 337.5 "/>
</svg>
