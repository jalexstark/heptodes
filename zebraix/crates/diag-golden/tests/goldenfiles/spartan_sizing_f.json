{
  "sizing_scheme": "SquareCenter",
  "base_width": 300.0,
  "base_height": 450.0,
  "axes_range": [
    -2.0,
    -1.5,
    2.0,
    1.5
  ],
  "drawables": [
    {
      "layer": 0,
      "drawable": {
        "Lines": {
          "start": [
            [
              -2.0,
              -1.5
            ]
          ],
          "end": [
            [
              -2.0,
              1.5
            ]
          ],
          "offsets": [
            [
              0.0,
              0.0
            ],
            [
              4.0,
              0.0
            ]
          ]
        }
      }
    },
    {
      "layer": 0,
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "start": [
            [
              -2.0,
              1.5
            ],
            [
              2.0,
              1.5
            ]
          ],
          "end": [
            [
              2.0,
              -1.5
            ],
            [
              -2.0,
              -1.5
            ]
          ],
          "offsets": [
            [
              0.0,
              0.0
            ]
          ]
        }
      }
    }
  ]
}
